internal/oci/runtime_oci.go

```diff
@@ -1755,13 +1755,20 @@ func (r *runtimeOCI) ProbeMonitor(ctx context.Context, c *Container) error {
 		return nil
 	}
 
+	if c.monitorProcessDead {
```

Comment:
should we also just unset monitorProcess and then the block above will run?

Reply:
Thanks, I changed to unset it.